teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < Person
+  def initialize(id:, age:, specialization:, name: 'Unknow', parent_permission: true)
+    super(id: id, age: age, name: name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    'true'
+  end
```

Comment:
- [ ] Remember that by convention, methods ended in "?" return boolean values, I think you misstyped the quotations here! Please consider removing them, at the moment you're returning a string. 🤓 🔧